src/averageAges.js

```diff
@@ -36,8 +52,31 @@ function calculateMenAverageAge(people, century) {
  *
  * @return {number}
  */
+let mother = '';
+
+function callback(arr) {
+  return (arr.name === mother);
+};
+
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  let peopleNew = [];
+
+  if (withChildren === true) {
```

Comment:
Consider using Array.prototype.filter() instead of a for loop to create 'peopleNew' array.